action.yml

```diff
@@ -9,12 +9,18 @@ inputs:
 
   all versioned paths:
     description: >
-      Whether to test all possible versioned base URLs:
-        /vMAJOR; /vMAJOR.MINOR; /vMAJOR.MINOR.PATCH
-      The latter two being optional base URLs according to the specification.
+      Whether to test the optional versioned URLs:
+        /vMAJOR.MINOR; /vMAJOR.MINOR.PATCH
+      If false, only the mandatory /vMAJOR url will be tested.
     required: false
     default: false
 
+  validate unversioned paths:
+      description: >
+        Whether to validate the base URL as a full OPTIMADE implementation.
```

Comment:
```suggestion
        Whether to validate the unversioned base URL as a full OPTIMADE implementation.
```

Reply:
I think this might need more careful wording, I'll come back to it tomorrow

Reply:
Have expanded on this description and added it to the README, let me know if you want something shorter.